code/analyze/ac/plots/de_PlotFFTs.m

```diff
@@ -309,18 +309,30 @@
           pwrLH = squeeze(ffts.orig2D(1,1,:,:)).*(pd<0).*(ffts.pals.an2D<=0.05);
 
           imgRH = ifft2(pwrRH); imgRH=imgRH(1:mSets.nInput(1), 1:mSets.nInput(2));
+          rMin = min(min(imgRH));
```

Comment:
You actually use that idiom below. Better to use a single idiom, so highly encourage the change here.